_posts/2019/2019-03-13-my-switch-from-js-to-ruby.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: My switch from JavaScript to Ruby
```

Comment:
Suggestions: (I've put the search results analysis in brackets)

* How I became fond of Ruby on Rails (as a JavaScript developer) (50)
* Amazing how fast things can be done in Rails (69)
* When I learnt that Ruby on Rails is magick (65)